stretch/virtualbox/Vagrantfile

```diff
@@ -59,6 +63,45 @@ Vagrant.configure('2') do |config|
     unless Vagrant.has_plugin?("vagrant-vbguest")
       raise 'VBGuest plugin required: vagrant plugin install vagrant-vbguest!'
     end
+
+    if ARGV[0] == 'up'
+      # Adds Virtual Disks
+      unless SubutaiConfig.get(:SUBUTAI_DISK).nil?
+        grow_by = SubutaiConfig.get_grow_by
+        generated_disk = SubutaiConfig.get(:_SUBUTAI_DISK)
+        disk_port = SubutaiConfig.get(:_SUBUTAI_DISK_PORT)
```

Comment:
What is this DISK_PORT about? Looks like you did a copy paste and kept this as _PORT? That's sloppy.

Reply:
While we attaching VM disk we need provide disk port.

Reply:
_SUBUTAI_DISK_PORT for saving last VM disk SATA port value. if we add new vm disk it gets value from that and increase by one.